setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 project_urls =
-    Documentation=https://pythonhosted.org/watchdog/
```

Comment:
@BoboTiG I grabbed this from the github description, so you should update it there

Reply:
Yes, I am working on it :)